re_data/command_line.py

```diff
@@ -197,4 +209,51 @@ def serve():
         httpd.server_close()
 
 
+@notify.command()
+@click.option(
+    '--start-date',
+    type=click.DateTime(formats=["%Y-%m-%d"]),
+    default=str((date.today() - timedelta(days=7)).strftime("%Y-%m-%d")),
+    help="Specify starting date to generate alert data, by default re_data will use 7 days ago for that value"
+)
+@click.option(
+    '--end-date',
+    type=click.DateTime(formats=["%Y-%m-%d"]),
+    default=str(date.today().strftime("%Y-%m-%d")),
+    help="""
+        Specify end date used in generating alert data, by default re_data will use current date for that.
+    """
+)
+@click.option(
+    '--webhook-url',
+    type=click.STRING,
+    required=True,
+    help="Incoming webhook url to post messages from external sources into Slack."
+         " e.g. https://hooks.slack.com/services/T0JKJQKQS/B0JKJQKQS/XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+)
+def slack(start_date, end_date, webhook_url):
```

Comment:
hm.. I would prefer `notify`. Something which in the future let us more easily add pagerduty/email so I imagine people would be calling `re_data notify slack` or `re_data notify pagerduty, `re_data notify email`

Reply:
that's currently how it works 🙂